src/main/fc/fc_rc.c

```diff
@@ -157,6 +166,9 @@ static void calculateSetpointRate(int axis)
     DEBUG_SET(DEBUG_ANGLERATE, axis, angleRate);
 
     setpointRate[axis] = constrainf(angleRate, -SETPOINT_RATE_LIMIT, SETPOINT_RATE_LIMIT); // Rate limit protection (deg/sec)
+
+    memcpy((uint32_t*)&setpointRateInt[axis], (uint32_t*)&setpointRate[axis], sizeof(float));
```

Comment:
What's the use of converting to pointer to `uint32_t` if `memcpy` accepts pointers to `void`?
Besides:
```cpp
*(float *) setpointRateInt = setpointRate[axis];
```